composer.json

```diff
@@ -31,10 +31,9 @@
     "drenso/phan-extensions": "^2.5"
   },
   "suggest": {
-    "magento/product-community-edition": "2.*",
-    "yotpo/module-review": "^2.9",
-    "ext-newrelic": "*",
-    "ext-code": "*"
+    "nosto/module-nostocmp": "Nosto Category Merchandising",
```

Comment:
Sorry for the delay in getting back to this. This package doesn't exist. Can you rename this to `nosto/nosto/module-nostotagging-cmp`.

Reply:
@hannunosto , updated!
Btw, maybe you change name of package nosto/module-nosto-msi to  nosto/module-nostotagging-msi for consintency?